src/ldap_helper.c

```diff
@@ -747,6 +739,25 @@ destroy_ldap_instance(ldap_instance_t **ldap_instp)
 	*ldap_instp = NULL;
 }
 
```

Comment:
The function is missing a comment. Please add one and describe what it does and under what conditions. Please see other Doxygen-style comments for inspiration.